Config.cmake.in

```diff
@@ -1,5 +1,7 @@
 @PACKAGE_INIT@
 
+set("@PROJECT_NAME@_INCLUDE_DIRS" "@CMAKE_INSTALL_FULL_INCLUDEDIR@")
+set("@PROJECT_NAME_UPPER@_INCLUDE_DIRS" "@CMAKE_INSTALL_FULL_INCLUDEDIR@")
```

Comment:
What is the reason to include both ?

Reply:
To be robust and projects like Eigen for instance only define the UPPER version